### DISCUSSION TITLE
not able to start the app

### DISCUSSION THREAD

**Discussion by tunguskar** [almarklein/timetagger#184] [Q&A]
Sorry for the stupid question. 

I installed everything like in the description. But I do not really get the point how to start the app. I got everything here:

```
C:\Users\xxx\timetagger>pip show timetagger
Name: timetagger
Version: 22.2.3
Summary: Tag your time, get the insight - an open source time tracker for individuals
Home-page: https://github.com/almarklein/timetagger
Author: Almar Klein
Author-email: almar.klein@gmail.com
License: GPL-3.0
Location: c:\programdata\python\python-3.9.8\lib\site-packages
Requires: asgineer, itemdb, jinja2, markdown, pscript, pyjwt, uvicorn
Required-by:
```

What do I have to do to start the app? I do not find the run.py file in this directory. sorry for noob question

**Comment by almarklein** [comment]
Hi, the `run.py` is on Github. Also see this post: https://timetagger.app/articles/selfhost/
